Update About the Docs page content (#1799)
hashicorp/terraform-website: content/source/docs/index.html.markdown

page_title: 'Documentation'
sidebar_current: 'docs-frontpage'
description: |-
  Documentation for Terraform, including Terraform CLI, Terraform Cloud, and Terraform Enterprise.
show_notification: false
---

# Terraform Documentation

Terraform is an infrastructure as code (IaC) tool that allows you to build, change, and version infrastructure safely and efficiently. This includes both low-level components like compute instances, storage, and networking, as well as high-level components like DNS entries and SaaS features.

<div class="container-fluid"><div class="row">
<div class="col-md-6 col-sm-12">

### Get Started

- Learn how Terraform [solves infrastructure challenges](/intro/index.html) and how it [compares to other tools and services](/intro/vs/index.html).
- Install Terraform and explore use cases with the [hands-on tutorials on HashiCorp Learn](https://learn.hashicorp.com/collections/terraform/aws-get-started).

### Manage Infrastructure

> - Describe infrastructure with [Terraform's configuration language](/docs/language/index.html) on various [providers](/docs/language/providers/index.html).

Reviewer: ```suggestion
- Describe infrastructure on various [providers](/docs/language/providers/index.html) with [Terraform's configuration language](/docs/language/index.html).
```

# Terraform Documentation

<div class="container-fluid"><div class="row">
<div class="col-md-6 col-sm-12">

### Get Started

- Learn how Terraform [solves infrastructure challenges](/intro/index.html) and how it [compares to other tools and services](/intro/vs/index.html).
- [Follow the tutorials on HashiCorp Learn](https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) to install Terraform, learn the Terraform workflow, and explore use cases.

### Manage Infrastructure

> - Use the [Terraform Configuration Language](/docs/language/index.html) to describe infrastructure.

Reviewer: ```suggestion
- Describe infrastructure in [Terraform Configuration Language](/docs/language/index.html).
```

Author: good call! I pushed a change that is very close to both of these with a minor change - see what you think!

> - Use the [Terraform CLI](/docs/cli/index.html) with the Terraform binary for command-line workflows.

Reviewer: ```suggestion
- Use the [Terraform Command Line Interface (CLI)](/docs/cli/index.html) to manage configuration, plugins, infrastructure, and state.
```

Reviewer: I think there's a world where we separate docs for the CLI and the binary itself, but we aren't there yet, and you can use many of the CLI commands with Terraform Cloud too :)

</div>

<div class="col-md-6 col-sm-12">

### Collaborate

> - [Terraform Cloud](/docs/cloud/index.html) helps teams use Terraform together, enabling version control, data sharing, and more.

Reviewer: ```suggestion
- [Terraform Cloud](/docs/cloud/index.html) helps teams use Terraform together, with version control, state sharing, and more.
```

> - [Terraform Enterprise](/docs/enterprise/index.html) offers a private instance of Terraform Cloud, with features like access control and governance.

Reviewer: ```suggestion
- [Terraform Enterprise](/docs/enterprise/index.html) is a self-hosted instance of Terraform Cloud.
```

Paid tiers of TFC also include these features

> ### Develop Plugins

Reviewer: Don't forget that you can also publish modules, which aren't plugins, on the Terraform Registry. I'm trying to think of how to retitle this section to keep it clean and also accurate, but drawing a bit of a blank... Maybe:

```suggestion
### Develop Modules and Plugins
```

- [Create a provider](/docs/extend/index.html) to allow Terraform to interact with a service.

> - [Publish your provider](/docs/registry/index.html) to the Terraform Registry to make it publicly available.

Reviewer: ```suggestion
- [Publish a provider or module](/docs/registry/index.html) to the Terraform Registry to make it publicly available.
```
